### PR TITLE
Update README for registory and repository

### PR DESCRIPTION
**Pull request by shhull**

### CODE REVIEW

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md [hunk 1]
```diff
@@ -118,6 +118,48 @@ Provision a Red Hat machine based on the following specifications, or using exis
...
+```sh
+sudo subscription-manager register --username <username> --password <password> --auto-attach
+```
+After registration, use the following command to enable ansible repository.
+```sh
+sudo subscription-manager repos --enable=ansible-2.8-for-rhel-8-s390x-rpms 
+```
+Install the packages from the repository in the system:
+```sh
+sudo yum install python3 ansible jq
```
Comment (gyohuangxin) [inline z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:133]: What's the version of ansible after installation?

Comment (shhull) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:133]:
```
[root@lynn-rhel82 yum.repos.d]# ansible --version
ansible 2.8.18
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.6/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.6.8 (default, Dec  5 2019, 15:45:24) [GCC 8.3.1 20191121 (Red Hat 8.3.1-5)]
```

Comment (gyohuangxin) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:133]: Thanks, but I only tested UPI with Ansible 2.11, have you ever done UPI seccessfully with Ansible 2.8.

File: z_infra_provisioning/cloud_infra_center/ocp_upi/quick-install.sh [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+stevedore>=2.0.1 # Apache-2.0
+EOF
+    sudo pip3 install -r requirements.txt python-openstackclient --ignore-installed
+    sudo alternatives --set python /usr/bin/python3
+}
+
+prepare_install() {
+# Default CIC password is dfltpass
+cat <<'EOF' >> password.txt 
+dfltpass
```
Comment (gyohuangxin) [inline z_infra_provisioning/cloud_infra_center/ocp_upi/quick-install.sh:65]: The default password of ICIC is the root user password of mgt node, not dfltpass.
How about make the password as a argument?

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md (hunk 1 above)
Comment (shhull) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/README.md]: Hello Xin,
I have tested all changes successfully in ansible 2.8.18.
```
[root@bastion2 ocp_upi]# ansible --version
ansible 2.8.18
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.6/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 3.6.8 (default, Mar 18 2021, 08:58:30) [GCC 8.4.1 20200928 (Red Hat 8.4.1-1)]
[root@bastion2 ocp_upi]# oc get nodes
NAME                       STATUS   ROLES    AGE    VERSION
host-172-26-105-63         Ready    worker   3m7s   v1.20.0+c8905da
host-172-26-105-68         Ready    worker   3m9s   v1.20.0+c8905da
openshift-5zb9r-master-0   Ready    master   40m    v1.20.0+c8905da
openshift-5zb9r-master-1   Ready    master   40m    v1.20.0+c8905da
openshift-5zb9r-master-2   Ready    master   40m    v1.20.0+c8905da
[root@bastion2 ocp_upi]#
```

File: z_infra_provisioning/cloud_infra_center/ocp_upi/quick-install.sh (hunk 2 above)
Comment (shhull) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/quick-install.sh:65]: yes, I have removed it.

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md [hunk 3]
```diff
@@ -113,11 +113,60 @@ Provision a Red Hat machine based on the following specifications, or using exis
...
 
+**Install:**
+
+Use the following command to register the system, then automatically associate any available subscription matching that system:
+```sh
+sudo subscription-manager register --username <username> --password <password> --auto-attach
+```
+After registration, use the following command to enable ansible repository.
+```sh
+sudo subscription-manager repos --enable=ansible-2.8-for-rhel-8-s390x-rpms
```
Comment (gyohuangxin) [inline z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:132]: One more question, why we use low version ansible-2.8 here?

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md (hunk 1 above)
Comment (shhull) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/README.md]: Verified on ZVM - 163 test stand:
```
[root@os015 ocp_upi]# oc get nodes
NAME                           STATUS   ROLES    AGE    VERSION
openshift-kl6xs-master-0       Ready    master   21m    v1.21.1+051ac4f
openshift-kl6xs-master-1       Ready    master   20m    v1.21.1+051ac4f
openshift-kl6xs-master-2       Ready    master   19m    v1.21.1+051ac4f
openshift-kl6xs-worker-64549   Ready    worker   9m7s   v1.21.1+051ac4f
openshift-kl6xs-worker-f2c77   Ready    worker   10m    v1.21.1+051ac4f
openshift-kl6xs-worker-f49c8   Ready    worker   11m    v1.21.1+051ac4f
[root@os015 ocp_upi]#
```

File: z_infra_provisioning/cloud_infra_center/ocp_upi/README.md (hunk 3 above)
Comment (shhull) [reply z_infra_provisioning/cloud_infra_center/ocp_upi/README.md:132]: I had not tested other version yet, maybe we could add one ansible upgrade version on 115.

**Comment by shhull** [comment]
Close this PR because some outdate changes.

**Pull request closed without merging**